images/peer/Dockerfile

```diff
@@ -26,13 +26,15 @@ WORKDIR $GOPATH/src/github.com/hyperledger/fabric
 FROM golang as peer
 ARG GO_TAGS
 RUN make peer GO_TAGS=${GO_TAGS}
+RUN make ccaasbuilder
 
 FROM peer-base
 ENV FABRIC_CFG_PATH /etc/hyperledger/fabric
 VOLUME /etc/hyperledger/fabric
 VOLUME /var/hyperledger
 COPY --from=peer /go/src/github.com/hyperledger/fabric/build/bin /usr/local/bin
 COPY --from=peer /go/src/github.com/hyperledger/fabric/sampleconfig/msp ${FABRIC_CFG_PATH}/msp
-COPY --from=peer /go/src/github.com/hyperledger/fabric/sampleconfig/core.yaml ${FABRIC_CFG_PATH}
+COPY --from=peer /go/src/github.com/hyperledger/fabric/sampleconfig/core.yaml ${FABRIC_CFG_PATH}/core.yaml
+COPY --from=peer /go/src/github.com/hyperledger/fabric/release/linux-amd64/bin/ccaas_builder/bin/ /opt/hyperledger/ccaas_builder/bin/
```

Comment:
Did you build the image locally and confirm it has the correct content and can be used?
e.g. `make docker-clean docker`

Reply:
I have done; and will repeat the exercise before final merge